Add T-ECDSA info to ETH integration pages (#2973)
dfinity/portal: docs/developer-docs/multi-chain/ethereum/using-eth/generating-addresses.mdx

## Canister ETH addresses

> To create an ETH address for your canister, first you will need to obtain a public key that will be used to sign your canister's Ethereum transactions using [threshold ECDSA chain-key signatures](docs/current/developer-docs/smart-contracts/encryption/t-ecdsa). You can get this key by calling the `ecdsa_public_key` API method of the management canister:

Reviewer: ```suggestion
To create an ETH address for your canister, first you will need to obtain an ECDSA public key associated to your canister. You can get this key by calling the `ecdsa_public_key` API method of the management canister:
```
